nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/java/org/apache/nifi/processors/script/TestExecuteJython.java

```diff
@@ -61,6 +61,34 @@ public void testReadFlowFileContentAndStoreInFlowFileAttributeWithScriptBody() t
         result.get(0).assertAttributeEquals("from-content", "test content");
     }
 
+    /**
+     * Tests a Jython script that references an outside python module
+     *
+     * @throws Exception Any error encountered while testing
+     */
+    @Test
+    public void testAccessModuleAndStoreInFlowFileAttributeWithScriptBody() throws Exception {
```

Comment:
Intellij says "Exception 'java.lang.Exception' is never thrown in the method" and suggests i should be removed.

Reply:
done.  Nice catch!